dynamic_rest/fields/fields.py

```diff
@@ -118,8 +122,15 @@ def bind(self, *args, **kwargs):
         parent_model = getattr(self.parent.Meta, 'model', None)
 
         remote = is_field_remote(parent_model, self.source)
+
+        def get_model_field():
+            if DJANGO110:
+                return parent_model._meta.get_field(self.source)
+            else:
+                return parent_model._meta.get_field_by_name(self.source)[0]
```

Comment:
that can be done with older releases. anyone need 1.7 will use older releases and newer release shouldn't continue to support unmaintained django versions.